src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -494,12 +494,14 @@ IExpression IMemberReferenceExpression.Instance
        
         protected override OperationKind ExpressionKind => OperationKind.MethodBindingExpression;
 
-        public override void Accept(IOperationVisitor visitor)
+        SyntaxNode IOperation.Syntax => this.Argument.Syntax;
```

Comment:
SyntaxNode for MethodBindingExpression should be the argument of DelegateCreationExpression.